src/common/components/wallet-ecency/index.tsx

```diff
@@ -281,20 +282,20 @@ export const WalletEcency = (props: Props) => {
 
     const isMyPage = activeUser && activeUser.username === account.name;
 
-    const dropDownConfig = {
-        history: history,
-        label: '',
-        items: [{
-            label: _t('points.transfer'),
-            onClick: toggleTransfer
-        }, {
-            label: _t('points.promote'),
-            onClick: togglePromote
-        }, {
-            label: _t('points.boost'),
-            onClick: toggleBoost
-        }]
-    };
+    // const dropDownConfig = {
+    //     history: history,
+    //     label: '',
+    //     items: [{
+    //         label: _t('points.transfer'),
+    //         onClick: toggleTransfer
+    //     }, {
+    //         label: _t('points.promote'),
+    //         onClick: togglePromote
+    //     }, {
+    //         label: _t('points.boost'),
```

Comment:
@adilrime17 could we not add the condition here that you added in the below?

Reply:
To make it consistent with the other component which is wallet-hive, I have added it below so developers can relate the code just by seeing it.
Let me know if you still want to change this.